DigitalOcean.API/Models/Responses/LoadBalancer.cs

```diff
@@ -6,87 +6,74 @@ public class LoadBalancer {
 		/// <summary>
 		/// A unique ID that can be used to identify and reference a Load Balancer.
 		/// </summary>
-		[JsonProperty("id")]
 		public string Id { get; set; }
 
 		/// <summary>
 		/// A human-readable name for a Load Balancer instance.
 		/// </summary>
-		[JsonProperty("name")]
 		public string Name { get; set; }
 
 		/// <summary>
 		/// An attribute containing the public-facing IP address of the Load Balancer.
 		/// </summary>
 		[JsonProperty("ip")]
-		public string Address { get; set; }
+		public string IpAddress { get; set; }
```

Comment:
Both `Domain` and `Interface` use the property name `IpAddress` for ip addresses.